Yoti.Auth.Sandbox/DocScan/Request/Resource/Extraction/TextExtractionResource.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Yoti.Auth.Sandbox.DocScan.Request.Resource.Extraction
+{
+    public class TextExtractionResource : SandboxPayload
```

Comment:
This should live with the integration tests (web client)

Reply:
Moved in ddbb848